acid/settings_reader.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
IMO this module should be named `utils.py` or `yaml_utils.py`
Because it hold only one function `read_yaml` and this has nothing in common with `settings`.